ci-operator/step-registry/cert-manager/custom-aggregated-cert/hypershift/cert-manager-custom-aggregated-cert-hypershift-commands.sh

```diff
@@ -0,0 +1,211 @@
+
+    mgmt oc patch hc -n clusters "$HC_NAME" --type merge -p "
+spec:
+  configuration:
+    apiServer:
+      servingCerts:
+        namedCertificates:
+        - servingCertificate:
+            name: $AGGREGATED_CERT_SECRET_NAME"
+
+    # Wait for kube-apiserver and oauth-openshift to restart
+    until (( $(mgmt oc get deployment -n "$HCP_NS" kube-apiserver -o jsonpath='{.metadata.generation}') > kas_generation )); do
+        sleep 15
+    done
+    mgmt oc rollout status deployment -n "$HCP_NS" kube-apiserver --timeout=15m
+    until (( $(mgmt oc get deployment -n "$HCP_NS" oauth-openshift -o jsonpath='{.metadata.generation}') > oauth_generation )); do
+        sleep 15
+    done
+    mgmt oc rollout status deployment -n "$HCP_NS" oauth-openshift --timeout=6m
+}
+
+function remove_kubelet_kubeconfig_cluster_ca() {
+    local pids_to_wait=()
+
+    for node in $(oc get node -o jsonpath='{.items[*].metadata.name}'); do
+        { timeout 90s oc debug node/"$node" -- chroot /host bash -c '
+# Wait for the debug pod to be ready
+sleep 60
+sed "/certificate-authority-data/d" /var/lib/kubelet/kubeconfig > /var/lib/kubelet/kubeconfig.tmp
+mv /var/lib/kubelet/kubeconfig.tmp /var/lib/kubelet/kubeconfig
+systemctl restart kubelet' || true; } &
+        pids_to_wait+=($!)
+    done
+    wait "${pids_to_wait[@]}"
+
+    # Nodes become unreachable
+    oc wait node --all --for=condition=Ready=Unknown --timeout=5m
+}
+
+function check_cert_issuer() {
+    local fqdn="$1"
+    local port="$2"
+    local issuer="$3"
+    local cert_issuers
+
+    cert_issuers="$(openssl s_client -connect "${fqdn}:${port}" -showcerts </dev/null 2>/dev/null | openssl x509 -noout -issuer)"
+    if ! grep -q "$issuer" <<< "$cert_issuers"; then
+        echo "Error: no ${fqdn}:${port} certificate issued by ${issuer}" >&2
+        return 1
+    fi
+}
+
+if [[ -f "${SHARED_DIR}/proxy-conf.sh" ]]; then
+    source "${SHARED_DIR}/proxy-conf.sh"
+fi
+
+# Timestamp
+export PS4='[$(date "+%Y-%m-%d %H:%M:%S")] '
+
+# Check clusterissuer readiness
+CLUSTERISSUER_NAME=cluster-certs-clusterissuer
+oc wait clusterissuer "$CLUSTERISSUER_NAME" --for=condition=Ready=True --timeout=0
+
+# Get CP service hostnames
+KAS_ROUTE_HOSTNAME="$(mgmt oc get hc -A -o jsonpath='{.items[0].spec.services[?(@.service=="APIServer")].servicePublishingStrategy.route.hostname}')"
+if [[ -z "$KAS_ROUTE_HOSTNAME" ]]; then
+    echo "Empty KAS route hostname, exiting" >&2
+    exit 1
+fi
+OAUTH_ROUTE_HOSTNAME="$(mgmt oc get hc -A -o jsonpath='{.items[0].spec.services[?(@.service=="OAuthServer")].servicePublishingStrategy.route.hostname}')"
+if [[ -z "$OAUTH_ROUTE_HOSTNAME" ]]; then
+    echo "Empty OAuth route hostname, exiting" >&2
+    exit 1
+fi
+HYPERSHIFT_EXTERNAL_DNS_DOMAIN="$(cut -d '.' -f 1 --complement <<< "$KAS_ROUTE_HOSTNAME")"
+
+# Create aggregated cert
+AGGREGATED_CERT_NAME=custom-ingress-cert
+AGGREGATED_CERT_SECRET_NAME=cert-manager-managed-ingress-cert-tls
```

Comment:
Considering the cert object specifies not only the ingress domain as dnsNames, it would be more precise to use `aggregated` instead of `ingress` for variables naming. WDYT?

Reply:
Thanks. Will address it later as a part of another pr